packages/authentication/src/types.ts

```diff
@@ -87,7 +87,9 @@ export interface AuthenticationStrategy {
    *
    * @param request - Express request object
    */
-  authenticate(request: Request): Promise<UserProfile | undefined>;
+  authenticate(
```

Comment:
Had a chat with @deepakrkris yesterday and I agree it's reasonable to allow returning a redirect route. Might be a breaking change.